integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/association/bytype/onetoone/ownedbycontained/AutomaticIndexingOneToOneOwnedByContainedLazyOnContainedSideIT.java

```diff
@@ -497,33 +498,40 @@ public static class ContainedEntity {
 		private Integer id;
 
 		@OneToOne(fetch = FetchType.LAZY)
+		@LazyGroup("containingAsIndexedEmbedded")
```

Comment:
Hmm, this is probably something worth mentioning in the ORM migration guide (https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc)?  my quick Cntrl+F didn't find anything 🙈

Reply:
Asked on Zulip: https://hibernate.zulipchat.com/#narrow/stream/132094-hibernate-orm-dev/topic/max_fetch_depth/near/338617431